### PR TITLE
Governance Polls 2024-11-04

### PR DESCRIPTION
**Pull request by blimpa**
@VoteWizard

### CODE REVIEW

File: governance/polls/Recentering Maker Brand - Novemeber 4, 2024.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+   0: Abstain
+   1: Keep the Sky brand as the backend protocol brand
+   2: Recenter the Maker brand
+   3: Recenter the Maker brand with a limited brand refresh
+start_date: 2024-11-04T16:00:00
+end_date: 2024-11-07T16:00:00
+---
+# Poll: Recentering Maker Brand - November 4, 2024
+
+The Governance Facilitators have placed a Governance Poll into the voting system in accordance with Atlas article [A.5.2 - Brand Identity - Recentering Maker Brand Polls](https://github.com/makerdao/next-gen-atlas/blob/0ad10bf7adb809299f2e35ca048b62708a3bbb6c/Sky%20Atlas/Sky%20Atlas.html#L8697BD). This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9.1_Operational_Weekly_Cycle-b189fa17-57a9-4d4e-9780-0ce4efd94211|0db30308) will be active for three days beginning on Monday, November 4 at 16:00 UTC.
```
Comment (VoteWizard) [inline governance/polls/Recentering Maker Brand - Novemeber 4, 2024.md:21]:
```suggestion
The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Accessibility Facilitators in accordance with Atlas article [A.5.2 - Brand Identity - Recentering Maker Brand Polls](https://sky-atlas.powerhouse.io/#A.5.1.1.3_Recentering_Maker_Brand_Polls-133f2ff0-8d73-80ba-add3-d43c59ec407b%7C8d5a16c0e403). This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9.1_Operational_Weekly_Cycle-b189fa17-57a9-4d4e-9780-0ce4efd94211|0db30308) will be active for three days beginning on Monday, November 4 at 16:00 UTC.
```

File: governance/polls/Recentering Maker Brand - Novemeber 4, 2024.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## Review
+
+The community can vote in this poll to express support or opposition to either one or none of the following options:
+
+- Option 1 - Keep the Sky brand as the backend protocol brand.
+- Option 2 - Recenter the Maker brand.
+- Option 3 - Recenter the Maker brand with a limited brand refresh.
+
+Please review the discussion [thread](https://forum.sky.money/t/weekly-atlas-edit-proposal-week-of-2024-10-28-0/25411) and the originating Atlas article [A.5.2 - Brand Identity - Recentering Maker Brand Polls](https://github.com/makerdao/next-gen-atlas/blob/0ad10bf7adb809299f2e35ca048b62708a3bbb6c/Sky%20Atlas/Sky%20Atlas.html#L8697BD) to help inform your position before voting.
```
Comment (VoteWizard) [inline governance/polls/Recentering Maker Brand - Novemeber 4, 2024.md:37]:
```suggestion
Please review the discussion [thread](https://forum.sky.money/t/weekly-atlas-edit-proposal-week-of-2024-10-28-0/25411) and the originating Atlas article [A.5.1.1.3 - Brand Identity - Recentering Maker Brand Polls](https://sky-atlas.powerhouse.io/#A.5.1.1.3_Recentering_Maker_Brand_Polls-133f2ff0-8d73-80ba-add3-d43c59ec407b%7C8d5a16c0e403) to help inform your position before voting.
```

File: governance/polls/Recentering Maker Brand - Novemeber 4, 2024.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+   0: Abstain
+   1: Keep the Sky brand as the backend protocol brand
+   2: Recenter the Maker brand
+   3: Recenter the Maker brand with a limited brand refresh
+start_date: 2024-11-04T16:00:00
+end_date: 2024-11-07T16:00:00
+---
+# Poll: Recentering Maker Brand - November 4, 2024
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Accessibility Facilitators in accordance with Atlas article [A.5.2 - Brand Identity - Recentering Maker Brand Polls](https://sky-atlas.powerhouse.io/#A.5.1.1.3_Recentering_Maker_Brand_Polls-133f2ff0-8d73-80ba-add3-d43c59ec407b%7C8d5a16c0e403). This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9.1_Operational_Weekly_Cycle-b189fa17-57a9-4d4e-9780-0ce4efd94211|0db30308) will be active for three days beginning on Monday, November 4 at 16:00 UTC.
```
Comment (VoteWizard) [inline governance/polls/Recentering Maker Brand - Novemeber 4, 2024.md:21]:
```suggestion
The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Accessibility Facilitators in accordance with Atlas article [A.5.1.1.3 - Brand Identity - Recentering Maker Brand Polls](https://sky-atlas.powerhouse.io/#A.5.1.1.3_Recentering_Maker_Brand_Polls-133f2ff0-8d73-80ba-add3-d43c59ec407b%7C8d5a16c0e403). This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9.1_Operational_Weekly_Cycle-b189fa17-57a9-4d4e-9780-0ce4efd94211|0db30308) will be active for three days beginning on Monday, November 4 at 16:00 UTC.
```

File: governance/polls/Recentering Maker Brand - Novemeber 4, 2024.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+The community can vote in this poll to express support or opposition to either one or none of the following options:
+
+- Option 1 - Keep the Sky brand as the backend protocol brand.
+- Option 2 - Recenter the Maker brand.
+- Option 3 - Recenter the Maker brand with a limited brand refresh.
+
+Please review the discussion [thread](https://forum.sky.money/t/weekly-atlas-edit-proposal-week-of-2024-10-28-0/25411) and the originating Atlas article [A.5.2 - Brand Identity - Recentering Maker Brand Polls](https://github.com/makerdao/next-gen-atlas/blob/0ad10bf7adb809299f2e35ca048b62708a3bbb6c/Sky%20Atlas/Sky%20Atlas.html#L8697BD) to help inform your position before voting.
+
+**Note that the Governance Portal does not currently support the specified winning criteria (described under **Outcome**) for this poll. In order to minimize confusion, this vote will have no winner displayed on the Governance Portal; however, the winning criteria will still apply at the end of it and the winner will be acknowledged in a subsequent forum post by the Governance Facilitators.**
```
Comment (VoteWizard) [inline governance/polls/Recentering Maker Brand - Novemeber 4, 2024.md:39]:
```suggestion
**Note that the Governance Portal does not currently support the specified winning criteria (described under **Outcome**) for this poll. However, the winning criteria will still apply at the end of it and the winner will be acknowledged in a subsequent forum post by the Governance Facilitators.**
```

File: governance/polls/SparkLend Ethereum & Base - Activate Spark Liquidity Layer - November 4, 2024.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
+---
+title: SparkLend Ethereum & Base - Activate Spark Liquidity Layer - November 4, 2024
```
Comment (VoteWizard) [inline governance/polls/SparkLend Ethereum & Base - Activate Spark Liquidity Layer - November 4, 2024.md:2]:
```suggestion
title: SparkLend Ethereum and Base - Activate Spark Liquidity Layer - November 4, 2024
```

File: governance/polls/SparkLend Ethereum & Base - Activate Spark Liquidity Layer - November 4, 2024.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-11-04T16:00:00
+end_date: 2024-11-07T16:00:00
+---
+# Poll: SparkLend Ethereum & Base - Activate Spark Liquidity Layer - November 4, 2024
```
Comment (VoteWizard) [inline governance/polls/SparkLend Ethereum & Base - Activate Spark Liquidity Layer - November 4, 2024.md:18]:
```suggestion
# Poll: SparkLend Ethereum and Base - Activate Spark Liquidity Layer - November 4, 2024
```

File: governance/polls/SparkLend Ethereum and Base - Activate Spark Liquidity Layer - November 4, 2024.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+| MAINNET_MIN_OPERATION_SIZE | 400,000 |
+| BASE_MIN_OPERATION_SIZE | 400,000 |
+| DEBT_CEILING_BUFFER | 10,000 |
+| USDC_MIN_BALANCE_BASE | 800,000 |
+| USDC_OPTIMAL_BALANCE_BASE | 800,000 |
+| USDC_MAX_BALANCE_BASE | 800,000 |
+
+Please review the discussion [thread](https://forum.sky.money/t/14-nov-2024-proposed-changes-to-spark-for-upcoming-spell/25466) to help inform your position before voting.
+
+For more information about the activation process and these parameters, please read the [_Spark ALM Planner Parameter Study_ forum post published by BAlabs](https://forum.sky.money/t/14-nov-2024-spark-alm-planner-parameter-study/25467).
```
Comment (VoteWizard) [inline governance/polls/SparkLend Ethereum and Base - Activate Spark Liquidity Layer - November 4, 2024.md:69]:
```suggestion
For more information about the activation process and these parameters, please read the [_Spark ALM Planner Parameter Study_ forum post published by BA Labs](https://forum.sky.money/t/14-nov-2024-spark-alm-planner-parameter-study/25467).
```

File: governance/polls/Recentering Maker Brand - Novemeber 4, 2024.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
+---
+title: Recentering Maker Brand - November 4, 2024
+summary: Signal your support or opposition to 1) keep the Sky brand as the backend protocol brand; or 2) recenter the Maker brand; or 3) recenter the Maker brand with a limited brand refresh.
```
Comment (MakerJanSky) [inline governance/polls/Recentering Maker Brand - Novemeber 4, 2024.md:3]: Support or opposition is not accurate, action required. Suggestion:

Signal your support of the following branding options to...

File: governance/polls/SparkLend Ethereum - Update Morpho DAI Vault Supply Caps - November 4, 2024.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to updating Morpho DAI vault Supply Caps as follows:
+
+- Increase [PT-sUSDe-26Dec2024 91.5% pool](https://morpho.blockanalitica.com/ethereum/markets/e37784e5ff9c2795395c5a41a0cb7ae1da4a93d67bfdd8654b9ff86b3065941c) Supply Cap from 100 million DAI to **250 million DAI**.
```
Comment (0xldr) [inline governance/polls/SparkLend Ethereum - Update Morpho DAI Vault Supply Caps - November 4, 2024.md:32]: Any docs for Supply Cap?

File: governance/polls/SparkLend Ethereum and Base - Activate Spark Liquidity Layer - November 4, 2024.md [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+
+## Review
+
+The community can vote in this poll to express support or opposition to activating the Spark Liquidity Layer on Sparklend Ethereum and Base with the parameters below.
+
+### Onchain Parameters
+
+#### Ethereum
+
+| Parameter | Max amount | Slope (tokens/day) |
```
Comment (0xldr) [inline governance/polls/SparkLend Ethereum and Base - Activate Spark Liquidity Layer - November 4, 2024.md:36]:
```suggestion
| Parameter | Max amount (tokens) | Slope (tokens/day) |
```

**Review by 0xldr — CHANGES_REQUESTED** [review]

**Review by MakerJanSky — CHANGES_REQUESTED** [review]
Minor items. Nice work, all.

File: governance/polls/Recentering Maker Brand - Novemeber 4, 2024.md (hunk 8 above)
Comment (VoteWizard) [inline governance/polls/Recentering Maker Brand - Novemeber 4, 2024.md:3]:
```suggestion
summary: Signal your support of the following branding options to 1) keep the Sky brand as the backend protocol brand; or 2) recenter the Maker brand; or 3) recenter the Maker brand with a limited brand refresh.
```

File: governance/polls/Recentering Maker Brand - Novemeber 4, 2024.md [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+
+**Note that the Governance Portal does not currently support the specified winning criteria (described under **Outcome**) for this poll. However, the winning criteria will still apply at the end of it and the winner will be acknowledged in a subsequent forum post by the Governance Facilitators.**
+
+## Outcomes
+
+**The winning criteria for this poll are as follows:**
+
+- The option with the most votes will be selected as the winning choice.
+- However, if Option 1 has the most votes individually, but the combined votes for Options 2 and 3 exceed those for Option 1, Option 2 will be declared the winner.
+
```
Comment (VoteWizard) [inline governance/polls/Recentering Maker Brand - Novemeber 4, 2024.md:55]:
```suggestion
- The winning option will direct the Maker/Sky workforce, with any required Atlas Edits being presented at a later date.

```

**Review by MakerJanSky — APPROVED** [review]
Nice work, all.

**Review by VoteWizard — APPROVED** [review]
Thanks, everyone! Great work!

**Pull request merged**